consensus/src/sync/light/validity_window.rs

```diff
@@ -360,8 +361,7 @@ impl<TNetwork: Network> LightMacroSync<TNetwork> {
                                 // We are complete so we emit the peer
```

Comment:
This comment is now a bit misleading as you don't move the peer directly into live sync anymore.